solver/scheduler.go

```diff
@@ -2,15 +2,14 @@ package solver
 
 import (
 	"context"
-	"encoding/csv"
```

Comment:
should we add a deny rule in https://github.com/moby/buildkit/blob/9aa07e61f874f13aaaf76d375d8c3cd92716b062/.golangci.yml#L50?

Reply:
For our use cases maybe but the new package is not a direct replacement. `encoding/csv` is for decoding big multi-line CSV files and is pretty good for that. If that is what you want to do you should keep using it and new package does not do the equivalent. Just our current usage of CSV parsing always deals with small single-line values instead.